src/components/Gradebook/index.jsx

```diff
@@ -433,13 +443,40 @@ export default class Gradebook extends React.Component {
                   open={this.state.modalOpen}
                   title="Edit Grades"
                   closeText="Cancel"
-                  body={(
+                  body={( // UI for this component
                     <div>
-                      <h3>{this.state.modalModel[0].assignmentName}</h3>
+                      <div>
+                        <div style={{ float: 'left', paddingRight: '49px' }}>Assignment: </div> <div>{this.state.modalAssignmentName}</div>
+                        <div style={{ float: 'left', paddingRight: '75px' }}>Student: </div> <div>{this.state.updateUserName}</div>
+                        <div style={{ float: 'left', paddingRight: '25px' }}>Original Grade: </div> <div>{this.state.originalGrade}</div>
+                        <div style={{ float: 'left', paddingRight: '25px' }}>Current Grade: </div> <div>{this.props.gradeOverrideCurrentPossibleGradedOverride}</div>
+                      </div>
                       <Table
-                        columns={[{ label: 'Username', key: 'username' }, { label: 'Current grade', key: 'currentGrade' }, { label: 'Adjusted grade', key: 'adjustedGrade' }]}
-                        data={this.state.modalModel}
+                        columns={[{ label: 'Date', key: 'date' }, { label: 'Grader', key: 'grader' },
+                        { label: 'Reason', key: 'reason' },
+                        { label: 'Adjusted grade', key: 'adjustedGrade' }]}
```

Comment:
Looks like we should move this to a CONSTANT

Reply:
What would be the benefit of constant? I found an example in the file on line 284 creating a const userInformationHeadingLabel. Is this the same thing?

Reply:
To make intention clearer, and to make the code easier to read. It calls out an assumption, and pulls it up to a different point in code where it might be reused/moved to where it could be even more widely reused if necessary.

E.g. we could validate that the props provided to our component doesn't contain data that doesn't fit in these columns. Or we could calculate the column labels & the corresponding propTypes from the same, larger data structure, reducing duplicating this list in multiple places.